### ISSUE TITLE
[Bug]: Need to update FBSDKCoreKit to 13.0.0

### ISSUE THREAD

**Issue by ariona** [oddbit/flutter_facebook_app_events#218]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

Can't update pod, since it need to update the FBSDKCoreKit to 13.0.0. I am using flutter_facebook_auth plugin which is using FBSDKCokire 13.0.0

```
[!] CocoaPods could not find compatible versions for pod "FBSDKCoreKit":
  In snapshot (Podfile.lock):
    FBSDKCoreKit (= 13.0.0, ~> 13.0.0)

  In Podfile:
    facebook_app_events (from `.symlinks/plugins/facebook_app_events/ios`) was resolved to 0.0.1, which depends on
      FBSDKCoreKit (~> 12.3.1)

    flutter_facebook_auth (from `.symlinks/plugins/flutter_facebook_auth/ios`) was resolved to 4.1.2, which depends on
      FBSDKCoreKit (~> 13.0.0)
```

### Expected Behavior

Plugin installed without error

### Platform

iOS

### Flutter Doctor

```shell
[✓] Flutter (Channel stable, 2.10.1, on macOS 12.2.1 21D62 darwin-x64, locale en)
    • Flutter version 2.10.1 at /Users/arionarian/Development/flutter-2.5
    • Upstream repository https://github.com/flutter/flutter.git
    • Framework revision db747aa133 (6 weeks ago), 2022-02-09 13:57:35 -0600
    • Engine revision ab46186b24
    • Dart version 2.16.1
    • DevTools version 2.9.2

[✓] Android toolchain - develop for Android devices (Android SDK version 31.0.0-rc3)
    • Android SDK at /Users/arionarian/Library/Android/sdk
    • Platform android-31, build-tools 31.0.0-rc3
    • Java binary at: /Applications/Android Studio.app/Contents/jre/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 11.0.10+0-b96-7281165)
    • All Android licenses accepted.

[✓] Xcode - develop for iOS and macOS (Xcode 13.2.1)
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • CocoaPods version 1.11.3

[✓] Chrome - develop for the web
    • Chrome at /Applications/Google Chrome.app/Contents/MacOS/Google Chrome

[✓] Android Studio (version 2020.3)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/6351-dart
    • Java version OpenJDK Runtime Environment (build 11.0.10+0-b96-7281165)

[✓] VS Code (version 1.65.2)
    • VS Code at /Applications/Visual Studio Code.app/Contents
    • Flutter extension version 3.36.0

[✓] Connected device (2 available)
    • Rian’s iPhone (mobile) • 4466a906da9032fcf7ce36d1ca597696cdc8add3 • ios            • iOS 14.4 18D52
    • Chrome (web)           • chrome                                   • web-javascript • Google Chrome 99.0.4844.83
    ! Error: Rian’s iPhone is busy: Fetching debug symbols for Rian’s iPhone. Xcode will continue when Rian’s iPhone is finished. (code -10)

[✓] HTTP Host Availability
    • All required HTTP hosts are available
```

**Comment by FrancoisBontemps** [comment]
Hi,

Only FBSDKCoreKit 12.3.1 is currently available for facebook_app_events and flutter_facebook_auth simultaneously.

add to your podfile : 
`pod 'FBSDKCoreKit', '12.3.1'`

and in your pubspec.yaml use : 
```
  facebook_app_events: 0.15.0
  flutter_facebook_auth: 4.1.0
```

this should solve your problem, but if you need FBSDKCoreKit 13 no matter what i don't have any solutions... both libs should upgrade their dependency

**Comment by zitob9** [comment]
> flutter_facebook_auth

dont work for me

`[!] CocoaPods could not find compatible versions for pod "FBSDKCoreKit":
  In Podfile:
    FBSDKCoreKit (= 13.0.0)

    facebook_app_events (from `.symlinks/plugins/facebook_app_events/ios`) was resolved to 0.0.1, which depends on
      FBSDKCoreKit (~> 12.3.1)

    flutter_facebook_auth (from `.symlinks/plugins/flutter_facebook_auth/ios`) was resolved to 4.1.2, which depends on
      FBSDKCoreKit (~> 13.0.0)

Specs satisfying the `FBSDKCoreKit (= 13.0.0), FBSDKCoreKit (~> 12.3.1), FBSDKCoreKit (~> 13.0.0)` dependency were found, but they required a higher minimum deployment target.`

**Comment by zakblacki** [comment]
Same problem !

**Comment by zitob9** [comment]
@DennisAlund doesn't work

**Comment by DennisAlund** [comment]
@zitob9 what doesn't work!? Please be helpful and respectful for an open source community project. 

Your comment is unspecific and without effort or respect for those who are helping to maintain and trying to serve you for free.

**Comment by zitob9** [comment]
@DennisAlund Sorry, I don't mean to offend. 

The solution that @FrancoisBontemps suggested does not work

**Comment by DennisAlund** [comment]
Thank you for clarifying. I apologize for slow progress, the suggested bump in PR is not considering the breaking changes and deprecations. 

My team has been having deadlines in other projects but we're trying to look at this by next week. 

We'd be extremely happy for help to walk through the breaking changes and deprecations that need to be considered for v13 bump

**Comment by SalsabeelaHasan** [comment]
Hi @DennisAlund, thanks to you and the team for your efforts.
Can you tell us when the new update will come out, accordingly I will plan to work on updating my application?

**Comment by zakblacki** [comment]
Same issue, still not resolved

**Comment by alihuseyin** [comment]
Same problem here 
```
facebook_app_events 0.18.1
flutter_facebook_auth 5.0.6
```

```
[!] CocoaPods could not find compatible versions for pod "FBSDKCoreKit":
  In Podfile:
    facebook_app_events (from `.symlinks/plugins/facebook_app_events/ios`) was resolved to 0.0.1, which depends on
      FBSDKCoreKit (~> 15.0.0)

    flutter_facebook_auth (from `.symlinks/plugins/flutter_facebook_auth/ios`) was resolved to 5.0.4, which depends on
      FBSDKLoginKit (~> 15.1.0) was resolved to 15.1.0, which depends on
        FBSDKCoreKit (= 15.1.0)
```
